content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Menginstal kubeadm
+content_type: task
+weight: 10
+card:
+  name: setup
+  weight: 20
+  title: Menginstal perkakas setup kubeadm
```

Comment:
let's make tool translation consistent with "alat"
```suggestion
  title: Menginstal alat persiapan kubeadm
```

Reply:
can replace all "perkakas" in this page with "alat"

Reply:
Alright, replaced in 0b74a869bea33a4eb86e43a89b5284bcd244444f